content/help/ops/security/authorization-permissive/index.md

```diff
@@ -4,14 +4,17 @@ description: Authorization is enabled, but requests make it through anyway.
 weight: 50
 ---
 If authorization checks are enabled for a service and yet requests to the service aren't being blocked, then
-its likely that authorization was not actually enabled successfully. You can do the following to verify:
+it's likely that authorization was not actually enabled successfully. You can do the following to verify:
 
-1. Check [this page](/docs/concepts/security/#authorization) to find out how to correctly enable Istio authorization policy. Use `istioctl` instead of `kubectl`, because `istioctl` enforces an additional validation check to prevent errors caused by typos.
+1. Check [this page](/docs/concepts/security/#enabling-authorization) to find out how to correctly enable
```

Comment:
To avoid non-descriptive link text, I suggest:
Check the \[enable authorization docs\] \(/docs/concepts/security/#enabling-authorization\)  to correctly enable